test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -49,6 +77,50 @@ public void parsesMetaNamespaceForm() throws IOException {
         assertEquals("Number of error messages", 0, parseResult.errorMessages.size());
     }
 
+    @Test
+    public void serializeAndRestore_MetaNamespaceFormInstance() throws IOException {
```

Comment:
Yes because I like the idea of having underscore in tests names because it's easier to read them and those test methods are never called by hand.
However all existing tests use camelCase so perhaps we shouldn't mix the conventions. 
At least in a single test class. I'll change this. 
Thanks!